bin/common.sh

```diff
@@ -35,11 +48,11 @@ hf() {
 hf_values() {
   [ "${VERBOSE-'false'}" = 'false' ] && quiet='--quiet'
   helmfile ${quiet-} -e "$CLOUD-$CLUSTER" -f helmfile.tpl/helmfile-dump.yaml build | grep -Ev $helmfileOutputHide | sed -e $replacePathsPattern |
-    yq read -P - 'releases[0].values[0]'
+    dyq read -P - 'releases[0].values[0]'
 }
 
 prepare_crypt() {
-  [[ -z "$GCLOUD_SERVICE_KEY" ]] && echo "Error: The GCLOUD_SERVICE_KEY environment variable is not set" && exit 2
+  [[ -z "$GCLOUD_SERVICE_KEY" ]] && echo "Error: The GCLOUD_SERVICE_KEY environment variable is not set" >&2 && exit 2
```

Comment:
Can we define and use `print_error` function?
```
function print_error() {
echo "Error: $1" >&2
}
```

Reply:
In `bin/otomi` STDERR is also printed like this. So to keep the same code style I think >&2 would be most solid. That way to keep the same code style we don't need to source `bin/common.sh` for the print_error function.

Reply:
Here I agree with @j-zimnowoda.

Reply:
Implemented